packages/ethereum-contracts/hardhat.config.ts

```diff
@@ -36,6 +36,10 @@ const config: HardhatUserConfig = {
             url: "http://0.0.0.0:8545/",
             chainId: 1337,
         },
+        avalanche: {
```

Comment:
what are these for actually, is hardhat used for anything network related?

Reply:
well, I guess I sometimes use it when trying to do something quick with hardhat console - but probably nothing other than that